gunrock/app/sm/sm_test.cuh

```diff
@@ -156,7 +154,7 @@ double CPU_Reference(util::Parameters &parameters, GraphT &data_graph,
 
       // Print out all mappings
```

Comment:
Please fix this comment. I don't think `callback` is  printing anything out here.